packages/storage-plus/README.md

```diff
@@ -384,7 +384,172 @@ fn demo() -> StdResult<()> {
 }
 ```
 
-## Indexed Map
+## IndexedMap
 
-TODO: we are working on a version of a map that manages multiple
-secondary indexed transparently. That work is coming soon.
+In cosmwasm-plus, there's currently one example of `IndexedMap` usage, in the `cw721-base` contract.
+Let's use it to illustrate `IndexedMap` definition and usage.
+
+### Definition
+
+```rust
+pub struct TokenIndexes<'a> {
+  // pk goes to second tuple element
+  pub owner: MultiIndex<'a, (Vec<u8>, Vec<u8>), TokenInfo>,
+}
+
+impl<'a> IndexList<TokenInfo> for TokenIndexes<'a> {
```

Comment:
This is rather annoyingly complex boilerplate.
I would consider a macro to generate this (one day)